R/MPI.R

```diff
@@ -348,10 +359,10 @@ fillPeaksChromPar <- function(arg) {
 
   lcraw <- xcmsRaw(arg$file, profmethod=params$prof$method, profstep = 0)
 
-  if(length(params$dataCorrection) > 1){
-    if(params$dataCorrection[i] == 1)
-      lcraw <- stitch(lcraw, AutoLockMass(lcraw))
-  }
+  # if(length(params$dataCorrection) > 1){
```

Comment:
Steffen,

I never really took the time to look at the parallel version of fillPeaks until recently and noticed that the stitch method has some issue in the way you implemented it in the fillPeaks method. Consequently I’ve removed this. There is also some evidence that the Waters cdf files ow take account of this. Further to this in the xcmsonline version I have access to the RAW files themselves which makes this much easier to implement. 
> Now done somewhere else ?
> 

Sort of - I may take the time to implement correctly in parallel fillPeaks.

Paul



> On May 6, 2015, at 1:16 PM, sneumann <notifications@github.com> wrote:
> 
> In R/MPI.R <https://github.com/sneumann/xcms/pull/13#discussion_r29797319>:
> 
> > @@ -348,10 +359,10 @@ fillPeaksChromPar <- function(arg) {
> >  
> >    lcraw <- xcmsRaw(arg$file, profmethod=params$prof$method, profstep = 0)
> >  
> > -  if(length(params$dataCorrection) > 1){
> > -    if(params$dataCorrection[i] == 1)
> > -      lcraw <- stitch(lcraw, AutoLockMass(lcraw))
> > -  }
> > +  # if(length(params$dataCorrection) > 1){
> Why is that removed ? Never worked ? Now done somewhere else ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sneumann/xcms/pull/13/files#r29797319>.
>